sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -49,8 +49,6 @@
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
     from azure.core.credentials import (
-        AzureKeyCredential,
-        AzureSasCredential,
         TokenCredential,
```

Comment:
do you need to do the same with `TokenCredential`?

Reply:
oh no I need to revert this actually sorry - they are all only used for typing purposes :)